src/home/components/HomeCard.js

```diff
@@ -16,20 +16,85 @@
  */
 import React from 'react';
 
-import _ from 'lodash/fp';
+import {
+  Box,
+  Card,
+  CardActions,
+  Divider,
+  Link,
+  Paper,
+  Stack,
+  Typography,
+} from '@mui/material';
 
-import { Card } from '@mui/material';
+import CardActionRouterLink from 'common/components/CardActionRouterLink';
+import RouterButton from 'common/components/RouterButton';
+import RouterLink from 'common/components/RouterLink';
 
-const HomeCard = props => (
+const HomeCard = ({
+  id,
+  title,
+  action,
+  image,
+  children,
+  isDefault,
+  useImg = true,
+  showActionsAsButtons = false,
+  contentBackgroundColor = 'blue.lite2',
+}) => (
   <Card
-    component="section"
-    {...props}
-    sx={{
-      display: 'flex',
-      ..._.getOr({}, 'sx', props),
-    }}
+    aria-labelledby={isDefault ? `${id}-default-title` : `${id}-list-title`}
```

Comment:
Issue: I think this component should not be aware of what `default` means, I think you can pass this value directly from the parent component, something like:
```suggestion
    aria-labelledby={labelledby}
```